### DISCUSSION TITLE
How to send a document and a file in one message

### DISCUSSION THREAD

**Discussion by lawbyte** [eternnoir/pyTelegramBotAPI#1995] [Q&A] [answered]
Please answer these questions before submitting your issue. Thanks!

1. What version of pyTelegramBotAPI are you using?
4.12.0
2. What OS are you using?
Debian
3. What version of python are you using?
3.9.2

I want to send a document and a message in one reply, can you guys please teach me how to do it?
Below are just an example, i want to do it in my bot so it will not be spam to other user because lot of reply will comes if i send one by one

![image](https://github.com/eternnoir/pyTelegramBotAPI/assets/35652825/870980ea-8db4-4f0d-b8a6-548504e64d5d)

**Comment by Badiboy** [comment, marked as answer]
https://pytba.readthedocs.io/en/latest/sync_version/index.html#telebot.TeleBot.send_document

**Reply by lawbyte** [reply]
thanks
